src/renderers/webxr/WebXRManager.js

```diff
@@ -190,12 +177,13 @@ function WebXRManager( renderer, gl ) {
 			session.addEventListener( 'squeezestart', onSessionEvent );
 			session.addEventListener( 'squeezeend', onSessionEvent );
 			session.addEventListener( 'end', onSessionEnd );
+			session.addEventListener( 'inputsourceschange', onInputSourceChange );
```

Comment:
Was there a reason for not adding the `inputsourceschange` event listener at this place?